.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Thank you for reporting an issue. The more information you can give us, the
+better the chance we can fix your problem.
+
+This issue tracker is for bugs and issues found within node-gyp,
+if you have an issue installing a specific module, please file an issue on
+that module's issue tracker (find the link with `npm issues modulename`).
+
+Please fill in as much of the template below as you're able.
+
+Node Version: output of `node -v` and `npm -v`
+Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
```

Comment:
It's  trade off since the output is
```
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.15063 N/A Build 15063
System Type:               x64-based PC
```
I'm not sure in `node` it's worth it since AFAIK there are way less version specific issues there. I think since `node-gyp` compiles it's way more version sensitive